src/frontend/NavigationStacks/AppStack.tsx

```diff
@@ -79,6 +81,8 @@ export const AppStack = createStackNavigator(
     LeaveProjectCompleted: LeaveProjectCompleted,
     AlreadyOnProj: AlreadyOnProj,
     AddToProject: AddToProjectScreen,
+    UnableToLink: UnableToLinkScreen,
+    JoinProjectQr: JoinProjectQrScreen,
```

Comment:
Does it make sense to name this slightly different than what's defined in the onboarding stack? Not totally sure myself but just noticed that there are now two routes pointing to the same component (although for different use cases)